.changeset/slimy-onions-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vygruppen/spor-react": patch
+---
+
+Updated styles for TextLink, and added dark mode support
```

Comment:
You should probably also mention the deprecation (and why it's deprecated).

Reply:
Good call. I've updated the comment to reflect our new vision for our components.

I'm bit unsure of the colors for dark mode with `variant=secondary`, so I will follow up on that before merging!